R/reportPrices.R

```diff
@@ -31,134 +31,164 @@
-                           "Price|Final Energy|Buildings|Heat (US$2005/GJ)"),
-                 setNames( dimSums(mselect(p_weights_FEprice_mkt * pm_FEPrice, all_enty1="fesos", emi_sectors = "build"), dim=3.3, na.rm = T)*tdptwyr2dpgj,
-                           "Price|Final Energy|Buildings|Solids (US$2005/GJ)")
-    ) 
-    
-    # FE Industry Prices
-    out <- mbind(out,
-                 setNames( dimSums(mselect(p_weights_FEprice_mkt * pm_FEPrice, all_enty1="feels", emi_sectors = "indst"), dim=3.3, na.rm = T)*tdptwyr2dpgj,
-                           "Price|Final Energy|Industry|Electricity (US$2005/GJ)"),
-                 setNames( dimSums(mselect(p_weights_FEprice_mkt * pm_FEPrice, all_enty1="fegas", emi_sectors = "indst"), dim=3.3, na.rm = T)*tdptwyr2dpgj,
-                           "Price|Final Energy|Industry|Gases (US$2005/GJ)"),
-                 setNames( dimSums(mselect(p_weights_FEprice_mkt * pm_FEPrice, all_enty1="feh2s", emi_sectors = "indst"), dim=3.3, na.rm = T)*tdptwyr2dpgj,
-                           "Price|Final Energy|Industry|Hydrogen (US$2005/GJ)"),
-                 setNames( dimSums(mselect(p_weights_FEprice_mkt * pm_FEPrice, all_enty1="fehos", emi_sectors = "indst"), dim=3.3, na.rm = T)*tdptwyr2dpgj,
-                           "Price|Final Energy|Industry|Liquids (US$2005/GJ)"),
-                 setNames( dimSums(mselect(p_weights_FEprice_mkt * pm_FEPrice, all_enty1="fehes", emi_sectors = "indst"), dim=3.3, na.rm = T)*tdptwyr2dpgj,
-                           "Price|Final Energy|Industry|Heat (US$2005/GJ)"),
-                 setNames( dimSums(mselect(p_weights_FEprice_mkt * pm_FEPrice, all_enty1="fesos", emi_sectors = "indst"), dim=3.3, na.rm = T)*tdptwyr2dpgj,
-                           "Price|Final Energy|Industry|Solids (US$2005/GJ)")
-    )
-   
-    
-    # apply lowpass filter to receive moving average prices
-    out.lowpass <- lowpass(out)
-    # add "Moving Avg" to variable name
-    getNames(out.lowpass) <- paste0(substr(getNames(out.lowpass), 
-                                           1,nchar(getNames(out.lowpass))-13),
-                                    "|Moving Avg",
-                                    substr(getNames(out.lowpass), 
-                                           nchar(getNames(out.lowpass))-12,
-                                           nchar(getNames(out.lowpass))))
-    # bind to output object
-    out <- mbind(out,out.lowpass)
-    
-    # add years before cm_startyear (temporary, can be adapted once prices only calculated after cm_startyear in REMIND code) 
-    out2 <- new.magpie(getRegions(out), getYears(vm_demFeSector), getNames(out), fill = NA)
-    out2[,getYears(out), ] <- out
-    
-    out <- out2
-    
-    
 
-}
-  
-  
-  
-  
-  
   ### end new price reporting
-  
-  
-  
-  
-  
-  
-  
-  
-  
-  
-  
-  
-  
-  
-  
-  
+
```

Comment:
I mean, I guess in the whole section below quite some variables like ``sebal`` and ``finenbal`` and stuff can be deleted in this old reporting part as they are not used anymore. We would need to clean this up another time maybe.

Reply:
As far as I see it is only the "Internal" biomass prices, the energy services prices and the carbon prices which are in this section of the "old" reporting now, right?